src/legacy/core_plugins/kibana/server/ui_setting_defaults.js

```diff
@@ -810,6 +810,18 @@ export function getUiSettingDefaults() {
       }),
       requiresPageReload: true,
     },
+    'theme:version': {
+      name: i18n.translate('kbn.advancedSettings.themeVersionTitle', {
+        defaultMessage: 'Theme version',
+      }),
+      value: 'v7',
+      type: 'select',
+      options: ['v7', 'v8'],
+      description: i18n.translate('kbn.advancedSettings.themeVersionText', {
+        defaultMessage: `Switch between the theme used for the current and next version of Kibana. A page refresh is required for the setting to be applied.`,
+      }),
+      requiresPageReload: true,
+    },
```

Comment:
Is this something we expect to ship in a version for users to try out? I partially dropped support for v8 in the dist but this component is ready for the distributable and will, as it's implemented, be available for users and break when it's changed to v8... Should I disable this uiSetting in the dist?

Reply:
I think that's actually a really good idea. So far the new theme has not been tested in Kibana and I'd be very wary of users turning it on right now and complaining of breaks. If it's easy enough to do, then yes please!

Reply:
It should be safe for people to use in any minor. There might be small breaks. I think as long as we label it as "beta" or something we're OK.

Reply:
Updated the v8 value to be `v8 (beta)`